services/ticket-service/.env.example

```
@@ -0,0 +1,16 @@
+# Application Configuration
+PORT=8085
+
+# Database Configuration (PostgreSQL)
+DB_HOST=localhost
+DB_PORT=5432
+DB_USER=admin
+DB_PASSWORD=password
+DB_NAME=authorizationdb
```

dotenv-linter (3.3.0)

[warning] [UnorderedKey] The DB_NAME key should go before the DB_PASSWORD key

(UnorderedKey)